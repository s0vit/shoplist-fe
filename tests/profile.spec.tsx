import { render } from '@testing-library/react';
import { expect, test } from 'vitest';

test('basic snapshot test', () => {
  // eslint-disable-next-line i18next/no-literal-string
  const rendered = render(<div>Hello Snapshot</div>);
  expect(rendered).toMatchSnapshot();
});
