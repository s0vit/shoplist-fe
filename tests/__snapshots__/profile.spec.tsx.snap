// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`basic snapshot test 1`] = `
<div>
  <div>
    Hello Snapshot
  </div>
</div>
`;

exports[`renders correctly 1`] = `[Function]`;
